GitCommands/Git/GitModule.cs

```diff
@@ -4153,6 +4153,43 @@ public string GetDescribe(ObjectId commitId)
             return output;
         }
 
+        public void LfsLock(IReadOnlyList<string> files)
+        {
+            if (files.Count == 0)
```

Comment:
The code of this method is the same than the Unlock. Could you factories to a common method taking the git parameters as an input?